R/cql-geom-predicates.R

```diff
@@ -86,14 +87,24 @@ cql_geom_predicate_list <- function() {
     "DWITHIN", "BEYOND", "BBOX")
 }
 
-sf_to_text_bbox <- function(x) {
+sf_text <- function(x) {
   if (!inherits(x, c("sf", "sfc", "sfg"))) {
     stop(paste(deparse(substitute(x)), "is not a valid sf object"),
          call. = FALSE)
   }
 
-  x = sf::st_bbox(x)
-  x = sf::st_as_sfc(x)
+  ## If too big here, drawing bounding
+  if(utils::object.size(x) > 5E5){
```

Comment:
I feel like 5MB is a bit on the small side...?

Reply:
Was trying to be conservative. I haven't found the threshold. I was actually wondering if there was a better way to deal with these hard coded values. I don't love the approach now. Would `options` or environment variables be appropriate here?